tools/lms-question-converters/README.md

```diff
@@ -0,0 +1,8 @@
+## Convert Learning Management System Questions to PrairieLearn
+
+Within this directory are a set of conversion tools for moving previously written questions
+in a different Learning Management System (LMS) to the question format for PrairieLearn. 
+
+- Craig Zilles: [LON-CAPA](lon-capa/)
+- Lawrence Angrave: [randexam](randexam/)
+- Lucas Wagner: [smartPhysics (FlipIt Physics)](smart-physics/)
```

Comment:
Let's not associate people's names with these. We don't do it elsewhere in the code. I'm fine to add names to the commit notes or somewhere like that. If we want to get really creative, you could actually make commits with these people as authors for each individual converter.

Reply:
Done. Switched the committers to Craig and Lawerence.